keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -290,6 +290,21 @@ internal class ClusterHandlerTests : JUnit5Minutests {
       }
     }
 
+    context("the cluster has active server groups with different app versions") {
+      before {
+        coEvery { cloudDriverService.activeServerGroup("us-east-1") } returns activeServerGroupResponseEast
+        coEvery { cloudDriverService.activeServerGroup("us-west-2") } returns activeServerGroupResponseWest.withOlderAppVersion()
+
+        runBlocking {
+          current(resource)
+        }
+      }
+
+      test("no event is fired indicating an app version is deployed") {
+        verify(exactly = 0) { publisher.publishEvent(ofType<ArtifactVersionDeployed>()) }
+      }
```

Comment:
Test we don't fire an event if the server groups in a cluster have different appVersions